_posts/2017-08-02-July.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Dev Diary (July 2017)"
+author: "Corey Montella"
+tags: []
+---
+
+We've had quite a busy July! Let's get to it:
+
+## v0.4 milestones
+
+In July we started using v0.4 to build internal projects, in order to find bugs in the runtime and test the limits of its performance. If you've hit performance walls with the v0.3 runtime, you'll be happy to know the Rust runtime is several orders of magnitude (10x - 100x) faster. Beyond the runtime itself, we've started fleshing out the tooling:
+
+- Server - The sever works like it did in v0.2, hosting an Eve instance listening on a websocket. We also include a static file server that hosts the output of your program.
```

Comment:
We gotta remember to put together some sort of program switcher before we ship it.